Library/Formula/gupnp-tools.rb

```diff
@@ -0,0 +1,22 @@
+require "formula"
+
+class GupnpTools < Formula
+  homepage "https://wiki.gnome.org/GUPnP/"
+  url "https://download.gnome.org/sources/gupnp-tools/0.8/gupnp-tools-0.8.10.tar.xz"
+  sha1 "a61b923c1b9f792cea6f805485d395212ef0b8af"
+
+  depends_on "pkg-config" => :build
+  depends_on "intltool" => :build
+  depends_on "gettext"
+  depends_on "gupnp"
+  depends_on "gupnp-av"
+  depends_on "gtk+3"
+  depends_on "gtksourceview3"
+  depends_on "ossp-uuid"
+
+  def install
+    system "./configure", "--disable-debug", "--disable-dependency-tracking",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+end
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!